src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java

```diff
@@ -29,49 +59,63 @@
     String name();
```

Comment:
Add comment

Reply:
It looks having a comment. The style is wrong?